### PR TITLE
Redirect HTTP to HTTPS if provided domain

### PR DESCRIPTION
**Pull request by McSinyx**
Fix GH-75.

### CODE REVIEW

File: src/acanban/__main__.py [hunk 1]
```diff
@@ -17,12 +17,19 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with Acanban.  If not, see <https://www.gnu.org/licenses/>.
 
+from hypercorn.middleware import HTTPToHTTPSRedirectMiddleware
 from hypercorn.trio import serve
+from hypercorn.typing import ASGIFramework
```
Comment (9a24f0) [inline src/acanban/__main__.py:22]: What does ASGI and HTTPToHTTPS... do

Comment (McSinyx) [reply src/acanban/__main__.py:22]: This is only for type annotation.  HTTP to HTTPS is to redirect every `http://acanban.ga/*` to `https://acanban.ga/*` (see linked issue).

**Review by Huy-Ngo — APPROVED** [review]
Since I don't know how this works I'll go the TAH way - merge then see what's broken.

**Comment by McSinyx** [comment]
Wise choice!  Me neither, I only follow the docs.

**Review by 9a24f0 — APPROVED** [review]

**Pull request merged**